websocket/services/expressLaneService.ts

```diff
@@ -0,0 +1,81 @@
+// check last saved from local cache or from pubnub history
+// query for this round from this timestamp (including)
+// publish query result to pubnub
+// (there may be duplicated messages for a specific timestamp, deduplicate by selecting the later messages(published later))
+// (because there may be edge cases where the timestamp has not fully been processed yet)
+
+import PubNub from "pubnub";
+import Ably from 'ably';
+import { ClickHouseClient } from "@clickhouse/client";
+import { PUBNUB_CHANNELS } from "@mevscan/shared/pubnub";
+import { config } from "@mevscan/shared/config";
+import { ABLY_CHANNELS } from "@mevscan/shared/ably";
+
+export interface ExpressLaneProfitData {
+    time: number;
+    profitUsd: number;
+    expressLanePrice: number;
+    currentRound: number;
+}
+
+export async function getExpressLaneProfitData(clickhouseClient: ClickHouseClient, lastStoredTime: number): Promise<ExpressLaneProfitData[]> {
+    const query = `
+        SELECT 
```

Comment:
I was thinking of just streaming the express lane transactions to the ably websocket and then aggregating the profit at the front end. the query would be:

```sql
SELECT * from mev.bundle_header where timeboosted = true
```

the frontend continuously updates the aggregation by adding up all the profits of given round. This way you would also get the transaction table streaming for free.